### PR TITLE
DT-1845,DT-1849: Update ICS2 requirements

### PR DESCRIPTION
**Pull request by HenrikHL**
[DT-1845](https://dcsa.atlassian.net/browse/DT-1845),[DT-1849](https://dcsa.atlassian.net/browse/DT-1849): Update ICS2 requirements

[DT-1845]: https://dcsa.atlassian.net/browse/DT-1845?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DT-1849]: https://dcsa.atlassian.net/browse/DT-1849?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ebl/v3/EBL_v3.0.0.yaml [hunk 1]
```diff
@@ -5355,14 +5350,13 @@ components:
...
           maxLength: 35
-          # Extended description of carrierBookingReference compared to DCSA_DOMAIN description
           description: |
             The associated booking number provided by the carrier for this `Consignment Item`.
           example: ABC709951
         descriptionOfGoods:
           type: array
           description: |
-            An array of lines accurately and properly describing the cargo being shipped in the container(s) as provided by the shipper.
+            A plain language description that is precise enough for Customs services to be able to identify the goods. General terms (i.e. 'consolidated', 'general cargo' 'parts' or 'freight of all kinds') or not sufficiently precise description cannot be accepted.
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:5359]: In the jira issue I see this part also in the description of DangerousGoods - Where the declarant provides the CUS code for chemical substances and preparations, Member States may waive the requirement of providing a precise description of the goods.

File: ebl/v3/EBL_v3.0.0.yaml [hunk 2]
```diff
@@ -4882,7 +4834,7 @@ components:
 
     NotifyPartyHBL:
       type: object
-      title: Notify Party
+      title: Notify Party (House B/L)
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:4837]: Don't we need the condition - Mandatory if House B/L isToOrder=true here?

File: ebl/v3/EBL_v3.0.0.yaml [hunk 3]
```diff
@@ -5458,7 +5451,7 @@ components:
         descriptionOfGoods:
           type: array
           description: |
-            An array of lines accurately and properly describing the cargo being shipped in the container(s) as provided by the shipper.
+            A plain language description that is precise enough for Customs services to be able to identify the goods. General terms (i.e. 'consolidated', 'general cargo' 'parts' or 'freight of all kinds') or not sufficiently precise description cannot be accepted.
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:5454]: same as above

File: ebl/v3/EBL_v3.0.0.yaml (hunk 2 above)
Comment (HenrikHL) [reply ebl/v3/EBL_v3.0.0.yaml:4837]: I will ask business

File: ebl/v3/EBL_v3.0.0.yaml (hunk 1 above)
Comment (HenrikHL) [reply ebl/v3/EBL_v3.0.0.yaml:5359]: `descriptionOfGoods` does not exist on DG. Not sure what you mean?

File: ebl/v3/EBL_v3.0.0.yaml (hunk 2 above)
Comment (HenrikHL) [reply ebl/v3/EBL_v3.0.0.yaml:4837]: HAHAHHA - but we have - it is just "outside" GitHub view
image

**Review by palatsangeetha — APPROVED** [review]
The changes looks good

**Pull request merged**